src/Tickets/Commerce/Utils/Currency.php

```diff
@@ -549,7 +549,108 @@ public static function get_default_currency_map() {
+			], 
+			'ALL' => [
+				'name'                  => __( 'Albanian lek (ALL)', 'event-tickets' ),
+				'symbol'                => 'L',
+				'decimal_point'         => '.',
+				'thousands_sep'         => ',',
+				'decimal_precision'     => 2,
+				'stripe_minimum_charge' => 115,
+			], 
+			'AMD' => [
+				'name'                  => __( 'Armenian dram (AMD)', 'event-tickets' ),
+				'symbol'                => 'AMD',
+				'decimal_point'         => '.',
+				'thousands_sep'         => ',',
+				'decimal_precision'     => 2,
+				'stripe_minimum_charge' => 455,
+			], 
+			'ANG' => [
+				'name'                  => __( 'Netherlands Antillean guilder (ANG)', 'event-tickets' ),
+				'symbol'                => '&fnof;',
+				'decimal_point'         => '.',
+				'thousands_sep'         => ',',
+				'decimal_precision'     => 2,
+				'stripe_minimum_charge' => 2,
+			], 
+			'AOA' => [
+				'name'                  => __( 'Angolan kwanza (AOA)', 'event-tickets' ),
+				'symbol'                => 'Kz',
+				'decimal_point'         => '.',
+				'thousands_sep'         => ',',
+				'decimal_precision'     => 2,
+				'stripe_minimum_charge' => 415,
+			], 
+			'ARS' => [
+				'name'                  => __( 'Argentine peso (ARS)', 'event-tickets' ),
+				'symbol'                => '&#036;',
+				'decimal_point'         => '.',
+				'thousands_sep'         => ',',
+				'decimal_precision'     => 2,
+				'stripe_minimum_charge' => 110,
+			], 
+			'AWG' => [
+				'name'                  => __( 'Aruban florin (AWG)', 'event-tickets' ),
+				'symbol'                => 'Afl.',
+				'decimal_point'         => '.',
+				'thousands_sep'         => ',',
+				'decimal_precision'     => 2,
+				'stripe_minimum_charge' => 2,
+			], 
+			'AZN' => [
+				'name'                  => __( 'Azerbaijani manat (AZN)', 'event-tickets' ),
+				'symbol'                => 'AZN',
+				'decimal_point'         => '.',
+				'thousands_sep'         => ',',
+				'decimal_precision'     => 2,
+				'stripe_minimum_charge' => 2,
+			], 
+			'BAM' => [
+				'name'                  => __( 'Bosnia and Herzegovina convertible mark (BAM)', 'event-tickets' ),
+				'symbol'                => 'KM',
+				'decimal_point'         => '.',
+				'thousands_sep'         => ',',
+				'decimal_precision'     => 2,
+				'stripe_minimum_charge' => 2,
+			], 
+			'BND' => [
+				'name'                  => __( 'Brunei dollar (BND)', 'event-tickets' ),
+				'symbol'                => '&#036;',
+				'decimal_point'         => '.',
+				'thousands_sep'         => ',',
+				'decimal_precision'     => 2,
+				'stripe_minimum_charge' => 2,
+			], 
+			'BZD' => [
+				'name'                  => __( 'Belize dollar (BZD)', 'event-tickets' ),
+				'symbol'                => '&#036;',
+				'decimal_point'         => '.',
+				'thousands_sep'         => ',',
+				'decimal_precision'     => 2,
+				'stripe_minimum_charge' => 3,
+			], 
 		] );
+
+		// Sort in alpha order by currency code.
+		ksort( $map );
```

Comment:
Sorting is expensive, do some sort of caching of the value after sorting so that it only sorts once.